lib/index.js

```diff
@@ -7,11 +7,7 @@ const Reporter = require('./reporter');
 const Signal = require('./signal');
 const defaults = {
   server: {
-    url: 'http://localhost:5000',
-    register: {
-      path: 'register',
-      auth: undefined
```

Comment:
we can do that... I added auth in this case so that the .filter method works and the register method is cleaned up.